cdk.context.json

```diff
@@ -6,5 +6,16 @@
     "us-east-1d",
     "us-east-1e",
     "us-east-1f"
+  ],
```

Comment:
not clear if we need this file. I think it should probably be in the .gitignore (sorry). I think it only applies to deployments done locally and wouldn't really make sense if used for staging or prod from seed.run CD.

Reply:
I'll remove the file and create another PR to update .gitignore